packages/twenty-server/src/database/typeorm/metadata/migrations/1725893697807-addIndexTypeColumnToIndexMetadata.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddIndexTypeColumnToIndexMetadata1725893697807
+  implements MigrationInterface
+{
+  name = 'AddIndexTypeColumnToIndexMetadata1725893697807';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TYPE metadata.INDEX_TYPES AS ENUM ('BTREE', 'GIN')`,
+    );
+
+    await queryRunner.query(`
+      ALTER TABLE metadata."indexMetadata" 
+      ADD COLUMN "indexType" metadata.INDEX_TYPES NOT NULL DEFAULT 'BTREE';
+    `);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
```

Comment:
Is this one useful since you are dropping the column below?

Reply:
good catch, this is legacy